src/v2/Components/FlashBanner.tsx

```diff
@@ -25,7 +22,22 @@ const messages: Record<FlashMessageKey, string | React.FC> = {
   expired_token: "Link expired. Resend verification email.",
 }
 
-export const FlashBanner: React.FunctionComponent<Props> = ({
+export const FlashBanner: React.FunctionComponent = () => {
+  const [messageCode, setMessageCode] = React.useState(null)
+  const isClient = typeof window !== "undefined"
+
+  React.useEffect(() => {
+    if (isClient) {
+      const query = qs.parse(window.location.search.slice(1))
+      query["flash_message"] && setMessageCode(query["flash_message"])
```

Comment:
Assuming the `snake_case` stuff is coming from gravity, yah? Otherwise we should try to stick to camel case where possibile.

Reply:
Force automatically snake cases url parameters for some pages, meaning that the flash message wouldn't appropriately display when it should. Some [Slack context](https://artsy.slack.com/archives/CP9P4KR35/p1590185759355700).

We'll avoid future pain by snake casing here.